requirements.txt

```diff
@@ -1,33 +1,14 @@
-astroplan
-astropy >= 3.0.0
-ccdproc
-codecov
-coveralls
-dateparser
+# These are packages that are NOT available via our
+# choice of conda repositories; there are less well
+# known repositories that do offer them, but that is
+# more of a security risk.
+
+# Python wrapper for ffmpeg.
 ffmpy
-gcloud
-google-cloud-storage
-matplotlib >= 2.0.0
-mocket
-numpy >= 1.12.1
-pycodestyle == 2.3.1
-pymongo >= 3.2.2
-pyserial >= 3.1.1
-pytest >= 3.4.0
-python_dateutil >= 2.5.3
-PyYAML >= 3.11
-pyzmq >= 15.3.0
-readline
-requests
-scikit_image >= 0.12.3
-scipy >= 0.17.1
 
-# Jupyter-notebook requires Tornado versions between 4 and 5. See:
-# https://stackoverflow.com/questions/49141525/install-jupyter-notebook-on-miniconda
-# https://github.com/ipython/ipython/issues/8249
-# https://github.com/jupyter/help/issues/324
+# Mock object support for Python testing.
+mocket
```

Comment:
Could potentially work around this. Since we have `requests` elsewhere now it might be cleaner to switch TheSkyX calls to `requests` and then use some of the mock tools provided for it, which are available.

Reply:
I'm trying to keep changes in this PR to a minimum (i.e. it will be large enough as is). I'd like to focus on just the switch to conda and 18.04. In fact, maybe I already shoved to much into this, and should extract the 18.04 (Mongodb) changes. Sigh. Lots of slow tests required.

Reply:
Agreed, don't want to do it for this PR, mostly just commenting on what I think we could eventually get rid of.